### DISCUSSION TITLE
字幕自动开启功能

### DISCUSSION THREAD

**Discussion by I-Have-A-Long-Name-Just-Like-This** [the1812/Bilibili-Evolved#4621] [功能建议 / Ideas]
需要一个自动开启视频字幕的功能

**Comment by Geeyun-JY3** [comment]
请善用搜索：#3745

**Comment by ChaXxl** [comment]
> 需要一个自动开启视频字幕的功能

+1, 我想要的是自动开启 AI 字幕的功能

**Reply by Geeyun-JY3** [reply]
作者都打了 `duplicate` 的标签了，就别在这 +1 了。
你的需求参见：https://github.com/the1812/Bilibili-Evolved/discussions/3745#discussioncomment-532226
